js/adapt-contrib-slider.js

```diff
@@ -256,11 +256,16 @@ define(function(require) {
 
         onNumberSelected: function(event) {
             event.preventDefault();
-            if (this.model.get('_isComplete')) return;
-            var index = parseInt($(event.currentTarget).attr('data-id')) - 1;
+
+            if (this.model.get('_isComplete')) {
+              return;
+            }
+
+            var index = this.getIndexFromValue(parseInt($(event.currentTarget).attr('data-id')));
+            var $scaler = this.$('.slider-scaler');
             this.selectItem(index);
-            this.animateToPosition(this.mapIndexToPixels(index));
-            this.setAltText(index + 1);
+            this.animateToPosition(this.mapIndexToPixels(index, $scaler));
+            this.setAltText(index);
```

Comment:
this need to be `(index + 1)` as before - otherwise the `aria-valuenow` label gets set to 1 less than the selected index (i.e. if I click on the number 3 then `aria-valuenow="2"`)